static/js/publisher/metrics/graphs/snapInstalls.js

```diff
@@ -0,0 +1,219 @@
+import Chart from "chart.js";
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { schemeCategory10 } from "d3-scale-chromatic";
+import { scaleOrdinal } from "d3-scale";
+
+class SnapInstalls extends Component {
+  constructor(props) {
+    super(props);
+
+    const metrics = [];
+    const error = false;
+    const loading = true;
+
+    this.state = {
+      metrics,
+      error,
+      loading
+    };
+  }
+
+  transformMetrics(json) {
+    let data = {
+      series: [],
+      max: 0,
+      buckets: json.buckets
+    };
+
+    const mapValues = (value, index) => {
+      if (data.max < value) {
+        data.max = value;
+      }
+
+      return {
+        x: json.buckets[index],
+        y: value
+      };
+    };
+
+    json.snaps.forEach(snap => {
+      // Grab the continued installs serie
+      const continuedInstalls = snap.series.filter(
+        serie => serie.name === "continued"
+      )[0];
+
+      const item = {
+        name: snap.name,
+        values: continuedInstalls.values.map(mapValues)
+      };
+
+      data.series.push(item);
+    });
+    return data;
+  }
+
+  getMetrics(snaps, token) {
+    // Limit snaps to 10 for performance reasons
+    const filteredSnaps = {};
```

Comment:
Not sure if it needs to be done in this PR, but would it make sense to load more snaps data later? So get snaps metrics in batches of 10 for example, and render them adding them to graph when they load?

Reply:
I agree we would need better handling of what/how many snaps to display, but possibly as a follow-up PR.